queries/futures/useGetFuturesPositionForMarkets.ts

```diff
@@ -22,10 +22,13 @@ const useGetFuturesPositionForMarkets = (
 	return useQuery<FuturesPosition[] | []>(
 		QUERY_KEYS.Futures.MarketsPositions(markets || []),
 		async () => {
+			if (!markets || (walletAddress && !isL2)) {
+				return [];
+			}
```

Comment:
short circuiting this here prevents this query from failing on mainnet due no Futures markets existing.

without this the following fails with:

```
const {
    contracts: { FuturesMarketData },
} = synthetixjs!;
```

<img width="1578" alt="Screen Shot 2022-04-07 at 3 05 10 PM" src="https://user-images.githubusercontent.com/5998100/162379676-aa3356c6-1bfc-4a4b-8ef6-7e115a4b51a7.png">